src/components/dialogs/confirm-dialog.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Box,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Stack,
+  Typography,
+} from "@mui/material";
+import CustomButton from "../common/button";
+import { ReactElement } from "react";
+import HelpOutlinedIcon from "@mui/icons-material/HelpOutlined";
+import React from "react";
+
+interface ConfirmDialogProps {
+  isOpen: boolean;
+  message: ReactElement;
+  onConfirm: () => void;
+  onClose: () => void;
+}
+
+const ConfirmDialog = (props: ConfirmDialogProps) => {
```

Comment:
For React component use Typescript there is a good way to approach is use `React.FC`

```typescript
const ConfirmDialog:React.FC<ConfirmDialogProps> = (props) => {
```